app/views/submission.scala.html

```diff
@@ -423,14 +433,15 @@ <h2 data-bind="text: abstract().title"></h2>
         <h3>Enter the Abstract Text</h3>
     </script>
     <script id="body-abstract-text-editor" type="text/html">
-        <textarea rows="10" class="form-control" id="text" maxlength="2000"
-        data-bind="value: editedAbstract().text, valueUpdate: 'afterkeydown'">
+        <textarea rows="10" class="form-control" id="text"
+                data-bind="value: editedAbstract().text, valueUpdate: 'afterkeydown'"
+                maxlength="@Model.getLimit[Abstract]("text")">
         </textarea>
         <div>
             <span>LaTeX-like equation typesetting is supported via MathJax. Use $$ and $ (inline).
-                <a target="_blank" href="http://meta.math.stackexchange.com/questions/5020/mathjax-basic-tutorial-and-quick-reference">Quick quide</a>
+                <a target="_blank" href="http://meta.math.stackexchange.com/questions/5020/mathjax-basic-tutorial-and-quick-reference">Quick guide</a>
             </span><br>
-            <span>Characters left: </span><span data-bind="text: editorTextCharactersLeft"></span>
+            <span data-bind="text: editorTextShowCharsLeft"></span><span data-bind="text: editorTextCharactersLeft"></span>
```

Comment:
Here would be the place where one could use the [visible binding](http://knockoutjs.com/documentation/visible-binding.html):

Something like this:
```html
<span data-bind="visible: editorTextCharactersLeft">
    Chars left: <span data-bind="text: editorTextCharactersLeft"></span>
</span>
```